srcpkgs/libspatialite/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'libspatialite'
 pkgname=libspatialite
 version=5.0.1
-revision=1
+revision=2
 build_style=gnu-configure
-hostmakedepends="pkg-config geos-devel"
+hostmakedepends="pkg-config geos-devel librttopo"
```

Comment:
why a library in hostmakedepends?

Reply:
I got this error, which went away after adding librttopo to hostmakedepends:

> checking for librttopo.h... yes
> checking for library containing rtt_AddLineNoFace... no
> configure: error: 'librttopo' (>= v.1.1.0) is required but it doesn't seem to be installed on this system. You may need to try re-running configure with a --disable-rttopo parameter.

Reply:
hm weird. and it works on cross too?

Reply:
Hmm, this was definitely the case then, but I can't reproduce this now so I'll remove it. Could this have been something with my build environment, like forgetting to clean after a failed build?

Reply:
yeah could be